### ISSUE TITLE
Planned data release: V11

### ISSUE THREAD

**Issue by jharenza** [AlexsLemonade/OpenPBTA-analysis#287]
#### What data file(s) does this issue pertain to?
V10 collapsed files were tables of genes removed, not the collapsed matrices, #248 

#### What release are you using?

V10

#### Put a link to the relevant section of the [OpenPBTA-manuscript](https://github.com/AlexsLemonade/OpenPBTA-manuscript/) here.
NA

#### Put your question or report your issue here.
Planned data to release:
- [ ] RNA-Seq collapsed FPKM files, #248 
- [ ] Putative oncogenic fusion TSV, #39

**Comment by jashapiro** [comment]
As long as there is an update planned, I will be able to provide updated consensus SNV files. (With MNVs included)

**Comment by jaclyn-taroni** [comment]
Related: #275

**Comment by jaclyn-taroni** [comment]
New consensus files from @jashapiro: https://open-pbta.s3.us-east-1.amazonaws.com/data/snv-consensus/snv-consensus-20191125.zip

**Comment by jharenza** [comment]
Re: putative oncogenic fusion (above), I am thinking we release the final prioritized list for the oncoprints, which would also become a supplemental table (I guess we add to the manuscript doc later?). cc: @jaclyn-taroni

**Comment by jaclyn-taroni** [comment]
To clarify, you are referring to this file: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/49acc98f5ffd86853fc70f220623311e13e3ca9f/analyses/fusion_filtering/results/PutativeOncogenicFusion.tsv correct @jharenza?

What are you going to name this file? I lean towards something like `pbta-fusion-putative-oncogenic.tsv`.

Regarding the new consensus mutation files, I think since we will probably put the `pbta-fusion-putative-oncogenic.tsv` directly in the release folder, we should make some changes to how the consensus mutation files are distributed.

The file structure of `snv-consensus-20191125.zip` is as follows:

```
.
├── README.md
├── consensus_mutation.maf.tsv
└── consensus_mutation_tmb.tsv
```

Could we rename the contents to:
`consensus_mutation.maf.tsv` -> `pbta-snv-consensus-mutation.maf.tsv` (+ possibly compress this file)
`consensus_mutation_tmb.tsv` -> `pbta-snv-consensus-mutation-tmb.tsv`

And then include these in the "top directory" of the release folder? I propose we move the information in the README file in the folder to probably 1) the release notes for this release and 2) the main README of the repository.  We may also want to include a Markdown file with the headers for these consensus files in `doc/format` as the first one is "MAF-like."

It may behoove us to create a section under [Data Formats](https://github.com/AlexsLemonade/OpenPBTA-analysis#data-formats) that talks specifically about these "derivative" data files (e.g., prioritized fusion list, consensus mutation files, collapsed expression matrices and maybe even the independent specimens files) and how we expect folks to use them.

I want to note that I'd prefer we really consider the documentation changes and wait until after the holiday to release v11 over getting this out today without documentation changes. We're also going to have to make a bunch of changes to how we generate the CI files before things consuming these files can get through reviewed and merged.

**Comment by jaclyn-taroni** [comment]
Moving the discussion over from #248 -- can we include the collapsed/summarized matrices and call them `pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds` and `pbta-gene-expression-rsem-fpkm-collapsed.polya.rds` please?

This is what I would expect based on reading the comments in code that generated them and we will ideally link the code in new section under `Data Formats` that I proposed in the comment above.

I think we can _remove_ the following files that were included in v10 from the download: `pbta-gene-expression-rsem-fpkm-collapsed_table.polya.rds` and `pbta-gene-expression-rsem-fpkm-collapsed_table.polya.rds` 

Sounds like @komalsrathi will be filing a pull request with some additional analyses today (https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/248#issuecomment-558377186). We can link to the notebook/script that gets added in `Data Formats` as well and direct folks to the `analyses/collapse-rnaseq/pbta-gene-expression-rsem-fpkm-collapsed_table.polya.rds` and `analyses/collapse-rnaseq/pbta-gene-expression-rsem-fpkm-collapsed_table.stranded.rds` files if they need more information.

**Comment by jharenza** [comment]
@jaclyn-taroni yes to the renaming and the putative oncogenic fusion file. @yuankunzhu is on it!

**Comment by jaclyn-taroni** [comment]
Great, thank you. If that pull request allows edits from maintainers, we can have someone on our side make the `doc/format` change + consensus mutation README additions.

**Comment by jharenza** [comment]
closed with #293
